### PR TITLE
megans functions

### PR DESCRIPTION
**Pull request by MeganHebert**

### CODE REVIEW

File: meg_func.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+def generate_productID():
+    num  = range(1,2000)
+    productID = random.sample(num,1)[0]
+    return productID
+
+def generate_name():
+    return fake.name()
+
+def generate_products():
```
Comment (Rainmaker0102) [inline meg_func.py]: Please make this into a dynamic provider following this example
```
def exampleProviderFunc():
    example_provider = DynamicProvider(
        provider_name = "example_provider",
        elements = ["my", "elements"],
    )
    return example_provider
```

**Review by Rainmaker0102 — CHANGES_REQUESTED** [review]
Good work! Please put these into the main.py file as well as fixing any additional comments

**Review by Rainmaker0102 — APPROVED** [review]
LGTM!

**Comment by TonyErazo** [comment]
LGTM

**Pull request merged**